package.json

```diff
@@ -146,8 +146,8 @@
     "build": "rollup -c",
     "build:release": "rollup -c --environment target:release",
     "build:debug": "rollup -c --environment target:debug",
-    "build:dts": "tsc src/index.js --outDir types --allowJs --declaration --emitDeclarationOnly",
-    "build:dts:extras": "tsc extras/index.js --allowJs --declaration --emitDeclarationOnly --outDir build/playcanvas-extras",
+    "build:dts": "tsc --project tsconfig.engine.json",
+    "build:dts:extras": "tsc --project tsconfig.extras.json",
```

Comment:
Try to keep this PR focused on the `LightComponent`. If you wanna revise the `package.json`, consider doing that in another PR.